.stylelintrc.json

```diff
@@ -42,16 +56,7 @@
     }],
 
     "declaration-empty-line-before": null,
```

Comment:
this is copied from aplus.

@phillimorland . this is now almost identical to the aplus config. I switched off three rules which are not easy to adapt imho. If okay for you I would add these lines to aplus.

Additionally I will adjust the `"max-nesting-depth": 3` in aplus which is set to `2` there.

Reply:
So, the rules in a+ are stricter? I think, then we should keep them stricter in a+. Especially not up the nesting depth as we want a lower one, but don't want to spent time on opin to also have that here.

Reply:
@khamui looks good, agree with @fuzzylogic2000 that we should keep it stricter in aplus maybe just add a comment here about the switched off rules so we have documentation about what we did. Also what lines do you mean when you say: 'If okay for you I would add these lines to aplus.' ?

Reply:
i meant the new switched off rules, but then i don't add them to aplus and just make add a comment here.

Reply:
@phillimorland just realized stylelint config is json, so no comment possible. maybe we can merge it and keep in mind to compare stylelint configs if we need to touch them again?